app/models/mention.rb

```diff
@@ -21,7 +21,7 @@ def send_email_notification
     user = User.find(user_id)
     return unless user.email.present? && user.email_mention_notifications
 
-    Mentions::SendEmailNotificationJob.perform_later(id)
+    Mentions::SendEmailNotificationWorker.perform_async(id)
```

Comment:
Since this is done in an [`after_create`](https://github.com/thepracticaldev/dev.to/pull/5549/files#diff-181fd4ada8f2799ca4ec9315d80401e6R12) callback we are going to need to update the callback to be `after_create_commit`

Reply:
My bad I did that change but I did not commit it :/

Reply:
@mstruve should be ok now :)